gcsweb/build/test.sh

```diff
@@ -20,15 +21,17 @@ set -o pipefail
 
 export CGO_ENABLED=0
 
-TARGETS=$(for d in "$@"; do echo ./$d/...; done)
+TARGETS=$(for d in "$@"; do echo "./$d/..."; done)
 
 echo "Running tests:"
+# shellcheck disable=SC2086
```

Comment:
Could explicitly make `$TARGETS` an array

Reply:
Outstanding

Reply:
Don't have arrays here :(

Reply:
can we add a comment about why we're doing this for the next person that comes along?